src/components/App.js

```diff
@@ -51,6 +63,11 @@ class App extends React.Component {
       );
     }
   }
-};
+  else{ 
```

Comment:
add another if condition here checking for a different variable say, isRulesVisible and render accordingly. We might add more components to be rendered and so, instead of an else an if statement for the particular component works. This way you render everything from App.js itself.